src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,98 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int MULTIPLIER = 2;
+    private int initialCapacity;
+    private int size;
+    private Node<K, V>[] array;
+
+    public MyHashMap() {
+        initialCapacity = DEFAULT_CAPACITY;
+        array = new Node[initialCapacity];
+        size = 0;
+    }
+
+    private int getIndex(K key) {
+        if (key == null) {
+            return 0;
+        }
+        return Math.abs(key.hashCode() % initialCapacity);
+    }
```

Comment:
Maybe This private method is best moved to the end.
(Optional) You can use here ternary operator.